### PR TITLE
doc: create scenario

### PR DESCRIPTION
**Pull request by pi-unnerup**
adding doc about scenario creation

### CODE REVIEW

File: docs/tasks-yaml-format.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ There are five top level fields in the spec:
 category: sample
 difficulty: Easy
 objective: Sample yaml
-kind: cp.simulator/scenario:<semver-version>
+kind: cp.simulator/scenario:<server-version>
```
Comment (s-irvine) [inline docs/tasks-yaml-format.md:11]: `semver` is correct I think. It's referring to semantic versioning for the tag

File: docs/create-scenario.md [hunk 2]
```diff
@@ -41,4 +38,24 @@ _Core Components_
...
+Additionally the `simulator/simulation-scripts/scenario/` directory can include a number of supporting bash scripts which can we executed on the master and worker nodes. These files help with particular setups on the nodes such as installing tooling, creating files or setting up users. The scripts are executed on startup and must have the following names:
+
+- _worker-any.sh_: runs on a randomly chosen worker node
+- _worker-1.sh_: runs on worker node 1
+- _worker-2.sh_: runs on worker node 2
+- _workers-every.sh_: runs on worker nodes 1 & 2
+- _nodes-every.sh_: runs on master and worker nodes 1 and 2
+- _master.sh_: runs on master
+
+Any other names will cause an error. See an example `[master.sh](http://master.sh)` script [here](https://github.com/kubernetes-simulator/simulator/blob/master/simulation-scripts/scenario/etcd-inverted-wedge/master.sh).
```
Comment (s-irvine) [inline docs/create-scenario.md:61]: Is this intended to literally print `[master.sh](http://master.sh)`?

**Review by s-irvine — CHANGES_REQUESTED** [review]

File: docs/tasks-yaml-format.md (hunk 1 above)
Comment (pi-unnerup) [reply docs/tasks-yaml-format.md:11]: semantic version - version?

File: docs/create-scenario.md (hunk 2 above)
Comment (pi-unnerup) [reply docs/create-scenario.md]: good spot

**Review by s-irvine — APPROVED** [review]
LGTM

**Pull request merged**